src/main/java/org/grizz/model/Planet.java

```diff
@@ -1,10 +1,19 @@
 package org.grizz.model;
 
+import com.google.common.collect.Sets;
 import lombok.Builder;
 import lombok.Data;
 
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.*;
+
 @Data
 @Builder
+@Document(collection = "planets")
 public class Planet {
+
     private String id;
```

Comment:
Are you assuming that planetNames will be unique? IMHO planets should be fetched by their coordinates in galaxy at most. I even do find useless fetching planets by ID in this case. But leave it as is for now.
And sure - we need planetName also, but not as an unique field